pkgs/misc/emulators/yuzu-mainline/default.nix

```diff
@@ -0,0 +1,27 @@
+{ stdenv, fetchgit, cmake, libressl, wrapQtAppsHook, SDL2, qtbase, python2 }:
+
+stdenv.mkDerivation {
+  pname = "yuzu-mainline";
+  version = "unstable-2020-04-13";
+
+  src = fetchgit {
+    url = "https://github.com/yuzu-emu/yuzu-mainline";
+    rev = "20c42dc9ea8bdc208d9d274350af2ff347876c76";
+    sha256 = "0s6vnvbyshbrl1xxiwdrxc2jl75jyjlf4kidl88c7pwj3m2gcwss";
+    leaveDotGit = true;
+    fetchSubmodules = true;
+  };
+
+  enableParallelBuilding = true;
+  nativeBuildInputs = [ cmake libressl wrapQtAppsHook ];
+  cmakeFlags = "-DCMAKE_BUILD_TYPE=Release";
+  buildInputs = [ SDL2 qtbase python2 ];
+
+  meta = with stdenv.lib; {
+    homepage = "https://yuzu-emu.org";
+    description = "An experimental open-source emulator for the Nintendo Switch";
+    license = licenses.gpl2;
```

Comment:
At the tail end of the license file there's some additional licenses listed for various assets included with yuzu. Unfortunately some of these assets are under CC BY-ND 3.0, which is a non-free license.

`license = with licenses; [ gpl2 cc-by-nd-30 cc0  ];`